Cubical/Homotopy/EilenbergSteenrod.agda

```diff
@@ -0,0 +1,64 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+module Cubical.Homotopy.EilenbergSteenrod where
+
+{-
+This module contains the Eilenberg-Steenrod axioms for ordinary reduced cohomology theories with binary additivity.
+The axioms are based on the ones given in Cavallo's MSc thesis (https://www.cs.cmu.edu/~ecavallo/works/thesis15.pdf) and Buchholtz/Favonia (2018)
+-}
+
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Pointed
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Equiv
+open import Cubical.HITs.Wedge
+open import Cubical.HITs.Pushout
+open import Cubical.HITs.Susp
+
+open import Cubical.Data.Nat
+open import Cubical.Data.Bool
+open import Cubical.Data.Sigma
+open import Cubical.Data.Int
+
+open import Cubical.Algebra.Group
+open import Cubical.Algebra.AbGroup
+open GroupEquiv
+open GroupHom
+
+private
+  variable
+    ℓ ℓ' ℓ'' : Level
+    A : Type ℓ
+    B : Type ℓ'
+
+cofib : (f : A → B) → Type _
+cofib f = Pushout (λ _ → tt) f
+
+cfcod : (f : A → B) → B → cofib f
+cfcod f = inr
+
+suspFun : (f : A → B) → Susp A → Susp B
+suspFun f north = north
+suspFun f south = south
+suspFun f (merid a i) = merid (f a) i
+
+contravar :  (H : Pointed ℓ → AbGroup {ℓ'}) → Type _
+contravar {ℓ = ℓ} H = {A B : Pointed ℓ} (f : A →∙ B) → AbGroupHom (H B) (H A)
```

Comment:
Why not inline this in the type of `f^*`? It doesn't seem like a very useful definition on its own and it doesn't pop up anywhere else